core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/ImageCatalogV4Controller.java

```diff
@@ -197,7 +198,7 @@ public ImageV4Response getImageFromDefaultById(Long workspaceId, @ResourceName S
 
     @Override
     @AccountIdNotNeeded
-    @DisableCheckPermissions
+    @InternalOnly
     public ImageV4Response getImageFromDefault(Long workspaceId, String type, String provider, String runtime) throws Exception {
 
         // FIXME: Implement this!
```

Comment:
And this.

Reply:
https://jira.cloudera.com/browse/CB-12624